### PR TITLE
Approve student request by mentor

### PR DESCRIPTION
**Pull request by ashatna**
fix get methods in Mentor models (return only mentor's students data)
add url to mentor urls
fix UI componen StudentsTabsList, StudentsTabsFilterBar
add services to StudentTabsListServices
fix UI components UserItem. UserList
add method get_mentors_topics to Topic models
add mentors_topics method to Topic views
create file topics_views_functions with helper function in utils
fix unittests fot mentor model and view

## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LEAR-63)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [ ] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/components/studentsTabsList/StudentsTabsList.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ export default class StudentsTabsList extends React.Component {
         const fromDate = this.state.filtersBar.fromDate;
         const toDate = this.state.filtersBar.toDate;
```
Comment (VitalyiKhlivnyi) [inline eventually/static/src/components/studentsTabsList/StudentsTabsList.js]: unnecessary empty line

File: eventually/static/src/components/studentsTabsList/studentsTabsListService.js [hunk 2]
```diff
@@ -24,6 +34,8 @@ const postStudentList = (studentId, chosenTopic) => {
     });
 };
 
+
+
```
Comment (VitalyiKhlivnyi) [inline eventually/static/src/components/studentsTabsList/studentsTabsListService.js]: two unnecessary empty lines

File: eventually/static/src/components/studentsTabsList/StudentsTabsList.js (hunk 1 above)
Comment (ashatna) [reply eventually/static/src/components/studentsTabsList/StudentsTabsList.js]: done!

File: eventually/static/src/components/studentsTabsList/studentsTabsListService.js (hunk 2 above)
Comment (ashatna) [reply eventually/static/src/components/studentsTabsList/studentsTabsListService.js]: done!

**Review by VitalyiKhlivnyi — DISMISSED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**